mixer/pkg/runtime/runtime.go

```diff
@@ -156,17 +156,17 @@ func (c *Runtime) processNewConfig() {
 	oldHandlers := c.handlers
 
 	newHandlers := handler.NewTable(oldHandlers, newSnapshot, c.handlerPool)
-
+	log.Infof("New handler table calculated:\n%s\n", newHandlers.String())
 	builder := compiled.NewBuilder(newSnapshot.Attributes)
 	newRoutes := routing.BuildTable(
-		newHandlers, newSnapshot, builder, c.defaultConfigNamespace, log.DebugEnabled())
+		newHandlers, newSnapshot, builder, c.defaultConfigNamespace, true)
 
 	oldContext := c.dispatcher.ChangeRoute(newRoutes)
 
 	c.handlers = newHandlers
 	c.snapshot = newSnapshot
 
-	log.Debugf("New routes in effect:\n%s", newRoutes)
+	log.Infof("New routes in effect:\n%s\n", newRoutes.String())
```

Comment:
Why does this and the other thing above need to be on by default? This is pollution in the user's log IMO.

Reply:
Better idea would be to put this behind a scope, and then selectively enable that scope.